core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -84,6 +87,10 @@ object ConsumerGroupCommand extends Logging {
     }
   }
 
+  def consumerGroupStatesFromString(input: String): List[ConsumerGroupState] = {
+    return input.split(',').map(s => ConsumerGroupState.parse(s.trim.toLowerCase.capitalize)).toSet.toList
+  }
```

Comment:
It seems that any inexistent states provided by the user will default to `UNKNOWN`. I wonder if it wouldn't be better to be stricter and raise an error directly when the states provided are invalid. What do you think?

Reply:
I agree, if invalid states are provided this should be treated as an error. I'll make the corresponding changes.